### DISCUSSION TITLE
1.6.0. pre-release broke unpacked

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#1056] [Help] [answered]
![image](https://user-images.githubusercontent.com/64666021/102265531-14507600-3ecc-11eb-8f82-68042579c17c.png)
When the unpacked version of 1.6.0. pre-release came, the extension was removed from my list and uploading does not work anymore. HELP PLEASE

**Comment by GrahamSH-LLK** [comment, marked as answer]
Fixed in latest commit https://github.com/ScratchAddons/ScratchAddons/commit/3731296919c1b9d01a85e3ce709b7d8e657f7d9c.

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/64666021/102266115-de5fc180-3ecc-11eb-8a65-525dc897ebfd.png)
Hope this works

**Reply by GrahamSH-LLK** [reply]
It should!
